### PR TITLE
snippets | snippets-stats | basket | careers docs/automation cleanup

### PR DESCRIPTION
**Pull request by metadave**
see https://github.com/mozmar/infra/issues/301

### CODE REVIEW

File: apps/snippets/README.md [hunk 1]
```diff
@@ -1,25 +1,55 @@
...
+In additional to the standard set of app variables (which can be found in our private
+repo), set the following values:
+
+- `AWS_STORAGE_BUCKET_NAME`
+- `CACHE_URL`
+- `DATABASE_URL`
+- `DEIS_DOMAIN`
+- `NEW_RELIC_APP_NAME`
+- `SITE_URL`
+- `STATSD_PREFIX`
```
Comment (glogiotatidis) [inline apps/snippets/README.md]: Add here `MEDIA_BUNDLES_URL` which is is in the format `<region>/bundles/`. 

So for frankfurt is `frankfurt/bundles/`

**Comment by glogiotatidis** [comment]
FTR IT owns the cloudfront CDN so we need to add a new origin there. I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1382211 for this

File: apps/snippets/README.md (hunk 1 above)
Comment (metadave) [inline apps/snippets/README.md]: fixed

**Comment by metadave** [comment]
merging so I can also update references to new repo name

**Pull request merged**